src/main/java/hudson/plugins/ec2/EC2Cloud.java

```diff
@@ -320,7 +320,7 @@ public void doAttach(StaplerRequest req, StaplerResponse rsp, @QueryParameter St
         StringWriter sw = new StringWriter();
         StreamTaskListener listener = new StreamTaskListener(sw);
         EC2AbstractSlave node = t.attach(id, listener);
-        Jenkins.getActiveInstance().addNode(node);
+        Jenkins.getInstance().addNode(node);
```

Comment:
Should use `Jenkins.get()` to avoid NPE.

Reply:
At the moment the code is mainly using the getInstance(), and the getActiveInstance() is deprecated. If we want to move to jenkins.get() we should handle in a separate "global clean up"commit

Reply:
yes, I am going to submit the patch for the problem reported by the findbugs

Reply:
It's amusing because `Jenkins.getInstance()` is a nullable method, but it wasn't an issue until they broke up the methods.